src/assets/scripts/client/navigationLibrary/NavigationLibrary.js

```diff
@@ -0,0 +1,93 @@
+import PositionModel from '../base/PositionModel';
+import FixCollection from './Fix/FixCollection';
+import StandardRouteCollection from './StandardRoute/StandardRouteCollection';
+
+/**
+ * @class NavigationLibrary
+ */
+export default class NavigationLibrary {
+    /**
+     * @constructor
+     * @for NavigationLibrary
+     * @param airportJson {object}
+     */
+    constructor(airportJson) {
+        /**
+         *
+         *
+         * @property _referencePosition
+         * @type {PositionModel}
+         * @default null
+         */
+        this._referencePosition = null
+
+        /**
+         *
+         *
+         * @property _sidCollection
+         * @type {StandardRoute}
+         * @default null
+         */
+        this._sidCollection = null;
+
+        /**
+         *
+         *
+         * @property _starCollection
+         * @type {StandardRoute}
+         * @default null
+         */
+        this._starCollection = null;
```

Comment:
I should also mention that a `StandardRoute` collection can contain any number of standard route definitions.  As long as the app also has fixes defined in the `FixCollection`.  The `StandardRoute` collections (`_sidCollection` and `_starCollection`) also reset on airport change along with the `FixCollection`.